utils/RapidproUtils.ts

```diff
@@ -0,0 +1,217 @@
+        });
+
+        return messages ? (messages.length > 0 ? logMessage : null) : null;
+    }
+
+    public async getContactMessages(token: string, after: string): Promise<IHttpResponse> {
+
+        const baseUrl = this.baseUrl;
+
+        const url = `https://${baseUrl}/api/v2/messages.json?contact=${token}&after=${after}`;
+
+        // TODO: RCAdmin checks if after is a string, but after analysis i dont think after can be anything else than a string, check if thats true
+
+        const contactMessagesResponse = await this.http.get(url, {
+            headers: this.getAuthorizationHeader(),
+        });
+
+        console.log(`Contact Messages Response: ${JSON.stringify(contactMessagesResponse)}`);
+
+        return contactMessagesResponse;
+
+    }
+
+    public getDateTime(message: any) {
+        const sentOn = new Date(message.created_on);
+        // TODO: check timezone effects
+        // sentOn.setHours(sentOn.getHours() - (sentOn.getTimezoneOffset() / 60))
+
+        const month = String(sentOn.getMonth()).padStart(2, '0');
+        const day = String(sentOn.getDay()).padStart(2, '0');
+        const year = String(sentOn.getFullYear()).padStart(2, '0');
+        const hour = String(sentOn.getHours()).padStart(2, '0');
+        const minutes = String(sentOn.getMinutes()).padStart(2, '0');
+        const seconds = String(sentOn.getSeconds()).padStart(2, '0');
+
+        const date = `${month}/${day}/${year} às ${hour}:${minutes}:${seconds}`;
+
+        return `[${date}]: `;
+    }
+
+    public getUserMessage(message: any) {
+        const attachments = message.attachments;
+
+        if (attachments && attachments.length > 0) {
+            const urls: Array<string> = [];
+            attachments.map( (e) => {
+                urls.push(e.url);
+            });
+            return urls;
+        }
+
+        return message.text.split('\n');
+
+    }
+
+    public formatUserMessage(line: string, message: any) {
+        return message.direction === 'out' ? line : '`' + line + '`';
+    }
+
+    public async broadcastMessages(messages: Array<any>, contacts: Array<string>) {
+        messages.map( (message) => {
+            const msgText = message.msg;
+
+            if (msgText.length === 0) {
+                // TODO: Check how to handle attachments
+                // msgText = getAttachmentUrl(message, contacts)
+            } else {
+                // TODO: Check how to handle emojis
+                // msgText = emojione.shortnameToImage(msgText)
+                const msgs = this.chunkString(msgText, this.MESSAGE_CHUNK_SIZE);
+
+                if (!msgs) {
+                    return;
+                }
+
+                msgs.map( async (msg) => {
+                    await this.broadcastMessage(msg, contacts);
+                });
+
+            }
+
+        });
+    }
+
+    public chunkString(str: string, length: number) {
+        return str.match(new RegExp('.{1,' + length + '}', 'g'));
```

Comment:
If user puts the following message:

```
ABCDE
FGHIJ
```

It will be interpreted as `"ABCDE\nFGHIJ"` in a string value, but the regex matches twice.

Expected: `[ "ABCDE\nFGHIJ" ]`
Actual: `[ "ABCDE", "FGHIJ" ]`

So, two requests would be executed instead of one.